### PR TITLE
bumping marathon 1.4.9 and updating 1.5.1.2

### PR DESCRIPTION
**Pull request by kensipe**
Bumping marathon 1.4.7 -> 1.4.8 which was a copy of the `/13` folder and update to proper version.   This change requires an update of the latest release as well which was a copy of the `/16` folder with an update to 1.5.1.2 which is the latest release.

### CODE REVIEW

**Comment by ichernetsky** [comment]
I still see `1.4.7` and `1.5.1.1` versions in the new files.

File: repo/packages/M/marathon/17/resource.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "images" : {
+    "icon-small" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-small.png",
+    "icon-medium" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-medium.png",
+    "icon-large" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-large.png"
+  },
+  "assets" : {
+    "container" : {
+      "docker" : {
+        "image" : "mesosphere/marathon:v1.4.7"
```
Comment (ichernetsky) [inline repo/packages/M/marathon/17/resource.json]: `1.4.9`?

File: repo/packages/M/marathon/18/resource.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "images" : {
+    "icon-small" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-small.png",
+    "icon-medium" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-medium.png",
+    "icon-large" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-large.png"
+  },
+  "assets" : {
+    "container" : {
+      "docker" : {
+        "image" : "mesosphere/marathon:v1.5.1.1"
```
Comment (ichernetsky) [inline repo/packages/M/marathon/18/resource.json]: `1.5.1.2`?

**Review by ichernetsky — CHANGES_REQUESTED** [review]

File: repo/packages/M/marathon/17/resource.json (hunk 1 above)
Comment (kensipe) [reply repo/packages/M/marathon/17/resource.json]: strange... I changed these... let me fix it

**Comment by kensipe** [comment]
Thanks @ichernetsky that was huge... I'm not sure how that happened.

**Review by ichernetsky — APPROVED** [review]

**Review by meln1k — APPROVED** [review]
LGTM

**Pull request closed without merging**